packages/www/docs/api-reference/session/list-recorded-sessions.mdx

```diff
@@ -25,17 +25,17 @@ curl -H 'authorization: Bearer {api-key}' \
     "record":true,
     "parentId":"nnnnnnnn-nnnn-nnnn-nnnn-nnnn",
     "recordingStatus":"ready",
-    "recordingUrl":"https://cdn.livepeer.com/recordings/aaaaaaaa-aaaa-aaaa-aaaa-aaaa/index.m3u8"},
-    "mp4URL":"https://mdw-cdn.livepeer.com/recordings/ccccccc-cccc-cccc-cccc-cccccccccccccc/2021_01_01_01_00_01-source.mp4",
+    "recordingUrl":"https://livepeercdn.com/recordings/aaaaaaaa-aaaa-aaaa-aaaa-aaaa/index.m3u8"},
+    "mp4URL":"https://mdw-livepeercdn.com/recordings/ccccccc-cccc-cccc-cccc-cccccccccccccc/2021_01_01_01_00_01-source.mp4",
```

Comment:
These regional URLs won't work like this. Maybe look for all resulting `-livepeercdn.com` to fix them back

Reply:
Right, so we keep the livepeer.com domain for the regional endpoints?